.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,32 @@
-### Summary
-<!-- Description of PR, with any special instructions for your reviewers. -->
 
-### Reason
-<!-- Why are you making this change? Can be a link to a Jira ticket, GH issue, 
-Trello card, etc. -->
+### Summary 
 
-### Testing
-<!-- How can your reviewers test your change? How did you test it? -->
+Description: <!-- What did you change and why? -->
+ 
+<!-- Include any supporting resources, e.g. link to a Jira ticket, GH issue, FTI, Slack, Aha, etc. -->
 
-<!-- (Optional) Include link to topic in Netlify preview after it's generated 
-(around 10mins after PR is created) -->
+Changelog entry, if applicable:
```

Comment:
Would this be a changelog of their changes in the PR? Or the changelog for the feature?

Reply:
Changelog for the feature. This seems to be confusing overall, so maybe I'll leave out anything about changelogs for now.